watson/watson.py

```diff
@@ -247,6 +248,13 @@ def start(self, project, tags=None):
                 )
             )
 
+        default_tags = self._default_tags(project)
+        if default_tags and not restart:
+            if tags is None:
+                tags = default_tags
+            else:
+                tags = tags + default_tags
```

Comment:
Ok, another version, which preserves order (replaces line 251 - 256 and depends on #114):

```
        if not restart:
            extra_tags = self.config.getlist('default_tags', project)
            tags = extra_tags + [tag for tag in (tags or [])
                                 if tag not in extra_tags]
```